pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -873,11 +877,19 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
                 player, playerLevel);
 
             // If the user was temp'd, show admins who temp'd the player.
-            if (tempLevel[player] == 1 || tempLevel[player] == 2) {
-                if (Player(playerid)->isAdministrator())
+                 
+                if (Player(playerid)->isAdministrator()) 
+                    {
+                  if (tempLevel[player] == 1 || tempLevel[player] == 2) 
                     format(message, sizeof(message), " %s {CCCCCC}(temp'd by %s){FFFFFF} (Id:%d) - {FF8E02}%s",
-                        Player(player)->nicknameString(), UserTemped[player], player, playerLevel);
-            }
+                        Player(player)->nicknameString(), UserTemped[player], player, playerLevel); 
+                    }
+                else {
+                    if (tempLevel[player] == 2) 
```

Comment:
This condition is caught in the if statement above. This code should never be reached as I'm reading it?

Reply:
I don't understand what you're trying to say.